### PR TITLE
test: Move fedora-atomic specific known issues into directory

### PR DESCRIPTION
**Pull request by stefwalter**
And remove the skip in check-journal, so we can let that fall
through to a known issue which will actually track the
problem.

### CODE REVIEW

File: test/check-journal [hunk 1]
```diff
@@ -178,13 +178,9 @@ ExecStart=/bin/sh -c 'for s in $(seq 10); do echo SLOW; sleep 0.1; done; sleep 1
...
-        if "222-10" in m.execute("rpmquery systemd || true"):
-            # HACK - https://bugzilla.redhat.com/show_bug.cgi?id=1292805
-            # --follow is broken again, so we need to switch pages explicitly
-            # and hope that the service has started sending out messages in the meantime...
-            b.go("#/?prio=*&start=oldest&service=nonexisting.service")
-            wait_journal_empty()
-            b.go("#/?prio=*&start=oldest&service=slow10.service")
+        b.go("#/?prio=*&start=oldest&service=nonexisting.service")
+        wait_journal_empty()
+        b.go("#/?prio=*&start=oldest&service=slow10.service")
```
Comment (dperpeet) [inline test/check-journal]: These three lines are actually the workaround. Their effect is that the page is manually reloaded by visiting a different page first. For a known issue, we can just delete them.

Comment (stefwalter) [inline test/check-journal]: Makes sense, thanks.

File: test/check-journal [hunk 2]
```diff
@@ -149,18 +149,9 @@ ExecStart=/bin/sh -c 'for s in $(seq 10); do echo SLOW; sleep 0.1; done; sleep 1
...
         m.execute("systemctl start log123")
-
-        if "222-10" in m.execute("rpmquery systemd || true"):
-            # HACK - https://bugzilla.redhat.com/show_bug.cgi?id=1292805
-            # --follow is broken again, so we need to switch pages explicitly
-            # and hope that the service has started sending out messages in the meantime...
-            b.go("#/?prio=*&start=oldest&service=nonexisting.service")
-            wait_journal_empty()
-            b.go("#/?prio=*&start=oldest&service=log123.service")
+        b.go("#/?prio=*&start=oldest&service=log123.service")
```
Comment (dperpeet) [inline test/check-journal]: we should still be on this page
https://github.com/cockpit-project/cockpit/pull/3391/files#diff-573b224077eb8056f85d23afe25f9a6bR109
What we could do is move those two lines beneath the function definition.

File: test/check-journal [hunk 3]
```diff
@@ -149,18 +149,9 @@ ExecStart=/bin/sh -c 'for s in $(seq 10); do echo SLOW; sleep 0.1; done; sleep 1
...
         wait_log123()
 
-        b.go("#/?prio=*&start=oldest&service=nonexisting.service")
-        wait_journal_empty()
```
Comment (dperpeet) [inline test/check-journal]: These two lines are actually part of the test to make sure that the data is still there if we change to a different page and come back.

**Comment by dperpeet** [comment]
I think you can just remove the `if "222-10" in m.execute("rpmquery systemd || true"):` blocks in `check-journal` and leave everything else as it is to make this work with the known issue.

**Comment by stefwalter** [comment]
Removed the check-journal stuff from here. The other ones are causing problems, so may as well get them in on their own.

**Pull request closed without merging**